package.json

```diff
@@ -10,12 +10,24 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@fortawesome/fontawesome-svg-core": "^6.4.2",
+    "@fortawesome/free-solid-svg-icons": "^6.4.2",
+    "@fortawesome/react-fontawesome": "^0.2.0",
```

Comment:
Other PRs, are using `heroicons-react`. Would you mind checking with them so that you can decide which one to use?